frontend/src/features/routes/RouteItem.tsx

```diff
@@ -118,10 +121,20 @@ function useClosestStop(to: Route): ClosestStop | undefined {
     distanceFromUser: formatMiles(
       geoDistanceToMiles(closestRouteStop.distance),
     ),
-    vanArrivalTime: estimateTime(closestRouteStopVan?.distance),
+    vanArrivalTime: formatSecondsAsMinutes(
+      closestRouteStopVan.inner.timeToNextStop,
+    ),
   };
 }
 
+function formatSecondsAsMinutes(seconds: number): string {
```

Comment:
We may want to use modern function syntax for consistency (I don't think we use this anywhere else?)

e.g.
`formatSecondsAsMinutes = (seconds: number) => {}`